pkg/action/action.go

```diff
@@ -113,6 +113,7 @@ func (cfg *Configuration) renderResources(ch *chart.Chart, values chartutil.Valu
 
 	if ch.Metadata.KubeVersion != "" {
 		if !chartutil.IsCompatibleRange(ch.Metadata.KubeVersion, caps.KubeVersion.String()) {
+			fmt.Fprintf(b, "WARNING: no --kube-version parameter passed, defaulting to %s\n", caps.KubeVersion.String())
```

Comment:
We can't print directly to stdout here I think. This package is used both by the helm cli and directly as an API.

The Action's Configuration's Log functionality should be used: https://github.com/helm/helm/blob/main/pkg/action/action.go#L97

Reply:
Got it, thanks a lot for the review and help, will definitely try to fix it soon!

Reply:
I think this should also only be output with the debug flag.